### PR TITLE
Videotron CMDB: Add all the required files with developer checklist and app-level code review

### PR DESCRIPTION
**Pull request by mmomin-crest**
#### Please check if your PR fulfills the following requirements:
- [x] Testing of all the changes has been performed (for bug fixes / features)
- [x] The readme.html has been reviewed and added / updated if needed (for bug fixes / features)
- [x] Use the following format for the PR description: `<App Name>: <PR Type> - <PR Description>`
- [x] Provide release notes as part of the PR submission which describe high level points about the changes for the upcoming GA release.
- [x] Verify all checks are passing.
- [x] Do NOT use the `next` branch of the forked repo. Create separate feature branch for raising the PR.
- [x] Do NOT submit updates to dependencies unless it fixes an issue.
- [x] Limit your pull request to one application and submit multiple pull requests if necessary.
#### Please check the type of change your PR introduces:
- [x] New App

## Release Notes (REQUIRED)
- Provide release notes as part of the PR submission which describe high level points about the changes for the upcoming GA release.

---
Thanks for contributing!

### CODE REVIEW

File: videotroncmdb.json [hunk 1]
```diff
@@ -0,0 +1,290 @@
...
+    "description": "This app integrates with the Videotron CMDB tool to perform lookups",
+    "type": "information",
+    "product_vendor": "Videotron",
+    "logo": "logo_videotroncmdb.svg",
+    "logo_dark": "logo_videotroncmdb_dark.svg",
+    "product_name": "CMDB",
+    "python_version": "3",
+    "product_version_regex": ".*",
+    "publisher": "Splunk",
+    "license": "Copyright (c) 2020-2021 Splunk Inc.",
```
Comment (ngaglani-crest) [inline videotroncmdb.json]: Change the copyright year to `2021` as it will be released for the first time.

File: videotroncmdb.json [hunk 2]
```diff
@@ -0,0 +1,290 @@
...
+    "app_version": "1.0.0",
+    "utctime_updated": "2020-07-14T10:41:41.182339Z",
+    "package_name": "phantom_videotroncmdb",
```
Comment (ngaglani-crest) [inline videotroncmdb.json:16]: * Add  bs4, requests, and all the dependent modules in pip_dependencies and add wheel files as well.
* Add bs4 and requests libraries in requirements.txt
Refer Tala for the same.

File: __init__.py [hunk 3]
```diff
@@ -0,0 +1,5 @@
+# File: __init__.py
+# Copyright (c) 2020-2021 Splunk Inc.
```
Comment (ngaglani-crest) [inline __init__.py]: Update Copyright year to `2021` for all the files.

File: videotroncmdb_connector.py [hunk 4]
```diff
@@ -0,0 +1,430 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# File: videotroncmdb_connector.py
+# Copyright (c) 2020-2021 Splunk Inc.
+#
+# SPLUNK CONFIDENTIAL - Use or disclosure of this material in whole or in part
```
Comment (ngaglani-crest) [inline videotroncmdb_connector.py]: Update license to Apache for all the files.

**Comment by ngaglani-crest** [comment]
Please add release notes in the [unrealesed.md](https://github.com/splunk-soar-connectors/videotroncmdb/blob/PAPP-21780_videotroncmdb/release_notes/unreleased.md) file.

File: videotroncmdb.json (hunk 1 above)
Comment (mmomin-crest) [reply videotroncmdb.json]: Changed the copyright year to 2021.

File: videotroncmdb.json (hunk 2 above)
Comment (mmomin-crest) [reply videotroncmdb.json:16]: Added bs4 and request wheel files and dependent modules in pip_dependencies and updated requirements.txt for same.

File: __init__.py (hunk 3 above)
Comment (mmomin-crest) [reply __init__.py]: Updated.

File: videotroncmdb_connector.py (hunk 4 above)
Comment (mmomin-crest) [reply videotroncmdb_connector.py]: Updated license to Apache.

File: readme.html [hunk 5]
```diff
@@ -0,0 +1,35 @@
+<!-- File: readme.html
+
+Copyright (c) 2018-2021 Splunk Inc.
```
Comment (ngaglani-crest) [inline readme.html]: Please update the copyright year to 2021

File: videotroncmdb.json [hunk 6]
```diff
@@ -0,0 +1,302 @@
...
+        }
+    ],
+    "pip_dependencies": {
+        "wheel": [
+            {
+                "module": "beautifulsoup4",
+                "input_file": "wheels/beautifulsoup4-4.9.1-py2-none-any.whl"
+            },
+            {
+                "module": "requests",
```
Comment (ngaglani-crest) [inline videotroncmdb.json:313]: Please add transitive dependencies (dependencies of requests and beautifulsoup) as `pip_dependencies` and also add their wheel files. We do not need to add those dependencies in requirements.txt

File: videotroncmdb.json [hunk 7]
```diff
@@ -0,0 +1,302 @@
...
+    "name": "Videotron CMDB",
+    "description": "This app integrates with the Videotron CMDB tool to perform lookups",
+    "type": "information",
+    "product_vendor": "Videotron",
+    "logo": "logo_videotroncmdb.svg",
+    "logo_dark": "logo_videotroncmdb_dark.svg",
+    "product_name": "CMDB",
+    "python_version": "3",
+    "product_version_regex": ".*",
+    "publisher": "Splunk",
```
Comment (dgopani-crest) [inline videotroncmdb.json]: This is a community app right? it should be `Splunk Community`

File: exclude_files.txt [hunk 8]
```diff
@@ -0,0 +1 @@
+.git*
```
Comment (dgopani-crest) [inline exclude_files.txt]: It is open-source app. we can remove exclude_files.txt

File: videotroncmdb.json (hunk 6 above)
Comment (mmomin-crest) [reply videotroncmdb.json:313]: Added transitive dependencies.

File: readme.html (hunk 5 above)
Comment (mmomin-crest) [reply readme.html]: Updated.

File: videotroncmdb.json (hunk 7 above)
Comment (mmomin-crest) [reply videotroncmdb.json]: Yes, Updated publisher as `Splunk Community`.

File: exclude_files.txt (hunk 8 above)
Comment (mmomin-crest) [reply exclude_files.txt]: Removed.

**Review by ngaglani-crest — APPROVED** [review]

**Review by tonyc-phantom — APPROVED** [review]

**Pull request merged**